swap_meet/vendor.py

```diff
@@ -0,0 +1,56 @@
+class Vendor:
+    pass
+    def __init__(self, inventory = None):
+        if inventory == None:
+            self.inventory= []
+        else:
+            self.inventory = inventory
+    def add(self, add_item):
```

Comment:
For readability I recommend placing a blank line between the end of one function and the beginning of the next.